windows_prepare_python.bat

```diff
@@ -49,23 +61,23 @@ if exist %dest_dir%\%python_full_name%.zip (
     if !errorlevel! neq 0 exit /b !errorlevel!
 )
 
-tar -xf %dest_dir%\%python_full_name%.zip -C %dest_dir%\%python_full_name%
+C:\Windows\System32\tar.exe -xf %dest_dir%\%python_full_name%.zip -C %dest_dir%\%python_full_name%
```

Comment:
This should not be needed

Reply:
Need once we have two options, msys tar and windows tar.